js-miniapp-bridge/src/types/responseTypes/interstitial/index.ts

```diff
@@ -0,0 +1,7 @@
+import { AdTypes } from '../../adTypes';
+/**
```

Comment:
Are there any specs around the `AdResponse` or is it just a placeholder declaration?